code/script.js

```diff
@@ -0,0 +1,94 @@
+const USER = "IdaAspen";
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const USER_INFO_URL = `https://api.github.com/users/${USER}`;
+
+const projectsContainer = document.getElementById("projects-container");
+const userContainer = document.getElementById("user-container");
+
+//get username and profile picture
+const getUserInfo = () => {
+  fetch(USER_INFO_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      userContainer.innerHTML = `
+      <div class="user-card">
+      <img src="https://avatars.githubusercontent.com/u/80949028?v=4"
```

Comment:
Indentation and formatting in ES6 HTML snippets can be quite tricky. Just wondering if the tags inside the "user-card" div should be indented one space to the right?

Reply:
Oh, innerHTML indentation is not my best friend at the moment. Thanks for pointing this out.